eqc/tcp_proto_eqc.erl

```diff
@@ -83,6 +83,12 @@ resolution() ->
 ttl() ->
     oneof([infinity, mtime()]).
 
+read_repair_opt() ->
+    oneof([0, 1, 2]).
+
+r_opt() ->
+    ?SUCHTHAT(I, int(), I >= 0 andalso (I band 16#FF) =:= I).
```

Comment:
`choose/2` might be better here `?SUCHTHAT` leads to 'non valid tests' that show as skipped plus is probably slower then `choose/2`.